tests/std/tests/P2674R1_is_implicit_lifetime/test.compile.pass.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <type_traits>
+

```

Comment:
Conventionally, MSVC STL's tests suite is `using namespace std;` (while others aren't).